roles/nodepool/defaults/main.yml

```diff
@@ -50,6 +50,13 @@ nodepool_labels:
     min-ready: 1
     providers:
       - name: cicloud
+- image: ubuntu-xenial
```

Comment:
Looks like this one started with - image: but the other started with - name:   Since they're dicts, I don't think it matters, but it does read odd to a human.